source/library/Patrol/Type/Envelope.hs

```diff
@@ -55,18 +45,16 @@ fromEvent dsn event =
 
 intoRequest :: (Catch.MonadThrow m) => Dsn.Dsn -> Envelope -> m Client.Request
 intoRequest dsn envelope = do
+  let uri =
+        Dsn.intoUri
```

Comment:
Actually I shouldn't be reusing `Dsn.intoUri` here because the DSN includes authorization information (username and maybe password) that we don't need or want in this request.